tests/tom-swifty-test/SwiftReflector/ProtocolConformanceTests.cs

```diff
@@ -289,5 +289,22 @@ get set
 			TestRunning.TestAndExecute (swiftCode, callingCode, "OK\n", platform: PlatformName.macOS);
 		}
 
+		[Test]
+		public void SomeProtocolAssocSubscriptGetSetParams ()
+		{
+			var swiftCode = @"
+public protocol Iterator6 {
+	associatedtype Elem
+	subscript (index: Elem) -> Elem {
+		get set
+	}
+}
+";
+			var printer = CSFunctionCall.ConsoleWriteLine (CSConstant.Val ("OK"));
+			var callingCode = CSCodeBlock.Create (printer);
+			TestRunning.TestAndExecute (swiftCode, callingCode, "OK\n", platform: PlatformName.macOS);
```

Comment:
that build but does not execute any of swift/binding code right ?
🐸

Reply:
Smoke test, like all the rest.